ta/pkcs11/src/pkcs11_helpers.h

```diff
@@ -20,6 +20,42 @@
 /* GPD TEE to PKCS11 status conversion */
 enum pkcs11_rc tee2pkcs_error(TEE_Result res);
 
+/*
+ * Return true if and only if attribute ID with companion attribute value
+ * size do match a valid attribute identifier.
+ *
+ * @attribute_id - Target PKCS11 attribute ID
+ * @size - Byte size of the attribute value, 0 if non-constant size
+ */
+bool valid_pkcs11_attribute_id(uint32_t attribute_id, uint32_t size);
+
+/*
+ * Return type attribute byte size if @attribute_id is the ID of a type
+ * attribute or 0 if not.
+ */
+size_t pkcs11_attr_is_type(uint32_t attribute_id);
+
+/* Return true if the object class is related to a type-on-class */
```

Comment:
Prefer "type-in-class" that is used at other places over "type-on-class".

Reply:
I'll fix